templates/fuzzing_bisection_without_regressed_by_needinfo.txt

```diff
@@ -1,2 +1,3 @@
-:{{ nickname }}, since this bug contains a bisection range, could you fill (if possible) the regressed_by field?
+{{ nicknames }}, since this bug contains a bisection range, could you fill (if possible) the regressed_by field?
```

Comment:
```suggestion
{{ nicknames }}, since this bug contains a bisection range, and you are the author of one of the changes in the range, could you fill (if possible) the regressed_by field?
```

(if there are multiple changes "and you are the author of one of the changes in the range", if there is a single change "and you are the author of the changes in the range")

Reply:
We could also say "if possible, could you fill the regressed_by field and investigate this regression?"

Reply:
We could needinfo the assignee (not necessarily an author of any of the changes); I will make the solution consider that.

Reply:
Yes, if there is an assignee we could needinfo the assignee instead of an author of one of the patches (but in most cases this tool will act on bugs where an assignee is not set yet, as the regression range is available soon after the bug is opened).

Reply:
Done in 676deace5d2ccb0e0741f68213d183ae9e055118



## Example (one author)

This bug contains a bisection range, and the `Regressed by` field is still not filled.

:pbone, since you are the author of the changes in the range, if possible, could you fill the `Regressed by` field and investigate this regression?

For more information, please visit [auto_nag documentation](https://wiki.mozilla.org/Release_Management/autonag#fuzzing_bisection_without_regressed_by.py).



## Example (multiple authors)

This bug contains a bisection range, and the `Regressed by` field is still not filled.

:botond and :nalexander, since you are the authors of the changes in the range, if possible, could you fill the `Regressed by` field and investigate this regression?

For more information, please visit [auto_nag documentation](https://wiki.mozilla.org/Release_Management/autonag#fuzzing_bisection_without_regressed_by.py).


## Example (assignee)

This bug contains a bisection range, and the `Regressed by` field is still not filled.

:pbone, if possible, could you fill the `Regressed by` field and investigate this regression?

For more information, please visit [auto_nag documentation](https://wiki.mozilla.org/Release_Management/autonag#fuzzing_bisection_without_regressed_by.py).